### PR TITLE
Update Dockerfile to centos 7.9

### PR DESCRIPTION
**Pull request by AlvaroVega**

### CODE REVIEW

File: docker/cygnus-ngsi/Dockerfile [hunk 1]
```diff
@@ -16,7 +16,7 @@
 # For those usages not covered by the GNU Affero General Public License please contact with iot_support at tid dot es
 #
 
-FROM centos:centos7.6.1810
+FROM centos:centos7.9.2009
```
Comment (fgalan) [inline docker/cygnus-ngsi/Dockerfile:19]: I think we have several Dockerfiles in this repo (four?). I'd suggest to do the same fix in all them.

**Review by fgalan — APPROVED** [review]
LGTM

**Review by IvanHdzC — APPROVED** [review]
LGTM

**Comment by fgalan** [comment]
I took a looooong time :) but finally travis finished. Merging.

**Pull request merged**